migrations/schema.sql

```diff
@@ -16,6 +16,42 @@ SET default_tablespace = '';
 SET default_with_oids = false;
 
 --
+-- Name: active_admin_comments; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+--
+
+CREATE TABLE active_admin_comments (
```

Comment:
it looks like your local copy of the database is out of sync -- you can either remove the tables listed here individually, until the schema.sql contains only your changes, or just drop your whole dev db and load a new version from production